hfc/fabric_network/gateway.py

```diff
@@ -0,0 +1,67 @@
+import logging
```

Comment:
Hi! Could you also add a line (`# SPDX-License-Identifier: Apache-2.0
`) at the top? Thanks!

Reply:
Will add the test cases in the future step.